src/ui/public/visualize/visualize.js

```diff
@@ -76,13 +76,20 @@ uiModules
 
       let applyClassNames = function () {
         let $visEl = getVisContainer();
+        const $spyEl = getter('.visualize-spy-container')();
```

Comment:
what do you think about defining getSpyElement in a way getVisContainer is defined just for consistency reasons ?